testing/ncids-js-testing/public/example-pages/nci-cta-strip.html

```diff
@@ -184,6 +183,9 @@ <h2>NCI CTA Strip Spanish</h2>
 					</ul>
 				</div>
 			</section>
+			<section class="usa-section">
```

Comment:
ew, just use an empty paragraph.  You wouldn't want this so show in the document outline.